en/development/testing.rst

```diff
@@ -657,6 +657,13 @@ Fixture directory. You can also load fixtures from CakePHP core, or plugins::
 Using the ``core`` prefix will load fixtures from CakePHP, and using a plugin
 name as the prefix, will load the fixture from the named plugin.
 
+.. note::
+
+    When using the plugin-prefix you have to add the plugins fixture folder path to composer.json (under autoload-dev).
```

Comment:
I am not sure this is required for prefixed plugins. It is required for any plugin that doesn't make its fixtures available through the autoloader defined in the plugin.

Reply:
okay, so what to do now? It took me Hours to find this, and there is no hint in the documentation. I'm pretty shure i'm not the only one having this problem. But because of your suggestion i've checked the cakedc/users-Plugin and opened an issue there: https://github.com/CakeDC/users/issues/676